### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by aniansson**
## Pivotal Tracker:
https://www.pivotaltracker.com/story/show/137600399

## Changes proposed in this pull request:
Updated changes and descriptions in file.

## What I have learned working on this feature: 
To not make changes directly on Github, to not branch off on Github, to not repeat mistake and to learn from mistakes. As well as updating and giving better descriptions in README files. And what should be included. 

Ready for review!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,33 +29,18 @@ And then
...
-
-### And coding style tests
+## Running the tests
 
-Explain what these tests test and why
+to run tests
 
 ```
-Give an example
+$rake
```
Comment (tochman) [inline README.md]: Add space
```
$ rake
```

File: README.md [hunk 2]
```diff
@@ -31,33 +29,18 @@ And then
...
 ```
 
 ## Deployment
 
-We are using Travis for continous integration and Heroku for deployment.
+We are using Travis for continous integration and Heroku for deployment
```
Comment (tochman) [inline README.md]: I would like you to add instructions on how the app can be deployed.

File: README.md [hunk 3]
```diff
@@ -31,33 +29,18 @@ And then
...
 
 ## Built With
```
Comment (tochman) [inline README.md:52]: I would like you to list the technologies used in this application.

**Review by tochman — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aniansson) [reply README.md]: What do you mean?

File: README.md (hunk 2 above)
Comment (aniansson) [reply README.md]: Do you mean entire instructions for ho to install etc?

**Review by tochman — APPROVED** [review]

**Pull request merged**